Classes/Twig/Environment.php

```diff
@@ -74,7 +78,7 @@ private function getTemplateStoragePath()
     {
         $settings = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['twig_for_typo3']);
 
-        $path = isset($settings['path']) ?  PATH_site . $settings['path'] : null ;
+        $path = isset($settings['rootTemplatePath']) ?  PATH_site . $settings['rootTemplatePath'] : null ;
```

Comment:
Shouldn’t using `\TYPO3\CMS\Core\Utility\GeneralUtility::getFileAbsFileName` be better? Then it is also possible to set `EXT:...` paths in the backend as well.